core/src/main/scala/kafka/server/ReplicaAlterLogDirsThread.scala

```
@@ -311,4 +330,13 @@ class ReplicaAlterLogDirsThread(name: String,
     }
   }
 
+  override protected def buildRemoteLogAuxState(partition: TopicPartition, currentLeaderEpoch: Int, fetchOffset: Long, epochForFetchOffset: Int, leaderLogStartOffset: Long): Long = {
+    // JBOD is not supported with tiered storage.
```

Should we just throw UnsupportedException?